README.md

```diff
@@ -200,21 +200,34 @@ Example:
 bitx.getOrder('BXHW6PFRRXKFSB4', function(err, result) {});
 ```
 
-### getTransactions(asset, [options, ]callback)
-GET https://api.mybitx.com/api/1/transactions
+### getTransactions(account_id, [options, ]callback)
+GET https://api.mybitx.com/api/1/accounts/:id/transactions
```

Comment:
Please use the same style of placeholder for variables as the rest of the docs i.e. `{accountId}`.